keps/prod-readiness/sig-node/2403.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2403
+stable:
+  approver: "@wojtek-t"
```

Comment:
Wojtek is over subscribed, I can look at this.

Reply:
No problem, do you want me to change the approver?

Reply:
yes please